src/spec.c

```diff
@@ -41,10 +41,13 @@ enum
 };
 
 static const char *bundle = NULL;
+static const char *fname = "config.json";
```

Comment:
Why not leave this NULL and then check if it is set?  Then you don't need the second boolean.

Reply:
fixed in the new version